sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/OctaneHttpResponse.java

```diff
@@ -15,77 +15,69 @@
 package com.hpe.adm.nga.sdk.network;
 
 import com.google.api.client.util.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.Charset;
 
 /**
- *
  * HTTP response.
  * Callers should call disconnect() when the HTTP response object is no longer needed. However, disconnect() does not have to be called if the response stream is properly closed.
- *
+ * <p>
  * Created by leufl on 2/11/2016.
  */
 public class OctaneHttpResponse {
 
     private final int statusCode;
-    private final InputStream inputStream;
+    private final byte[] content;
+
+    private static final Logger logger = LogManager.getLogger(OctaneHttpResponse.class.getName());
     private final Charset responseCharset;
 
     public OctaneHttpResponse(int statusCode, InputStream inputStream, Charset responseCharset) {
         this.statusCode = statusCode;
-        this.inputStream = inputStream;
         this.responseCharset = responseCharset;
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        try {
+            IOUtils.copy(inputStream, out);
```

Comment:
This will break all attachment functionality... reading from that input stream will render it unusable again. For attachments you'd use getInputStream, but that input stream will be closed already